website/server/models/user/methods.js

```diff
@@ -91,3 +98,17 @@ schema.methods.addComputedStatsToJSONObj = function addComputedStatsToUserJSONOb
 
   return statsObject;
 };
+
+schema.methods.cancelSubscription = async function cancelSubscription () {
+  let plan = this.purchased.plan;
+
+  if (plan.paymentMethod === amazonPayments.constants.PAYMENT_METHOD_AMAZON) {
```

Comment:
why is amazon's constant called PAYMENT_METHOD_AMAZON while paypal's and stripe PAYMENT_METHOD?